docs/docs/concepts/tool_calling.mdx

```diff
@@ -0,0 +1,133 @@
+# Tool Calling
+
+## Motivation 
+
+Many AI applications interact directly with humans (e.g., chatbots). In these cases, it is approrpiate for models to respond in natural langague. 
+
+But what about cases where we want a model to interact *directly* with another system (e.g., a databases or an API)?
+
+These systems often have a particular input schema (e.g., APIs have a required payload structure), making natural language inapporpiate for use.
+
+This motivates the concept of tool calling:
+
+* Tools (e.g., a databases or an API) can be bound to models.
+* Models can respond in the input schema required to run the tool.
+
+## Basic Usage
+
+:::info
+You will sometimes hear the term `function calling`. We use this term interchangeably with `tool calling`. 
+:::
+
+When using tools, we need to specify two things:
+
+* A model that supports tool calling.
+* Tools that the model can call.
+
+#### Model
+
+While tool calling is not universal, many model providers support it.
+
+See our [model integration page](/docs/integrations/chat/) for a list of models that support tool calling.
+
+LangChain provides a standardized interface for tool calling that is consistent across different models.
+
+`ChatModel.bind_tools()` is a method for specifying which tools are available for a model to call. 
+
+If a model has been initialized as `llm` without tools, we can bind tools to it as a list:
+
+```python
+llm_with_tools = llm.bind_tools([tool])
+```
+
+#### Tool
+
+Of course, need to specify a `tool` that the model can call.
+
+In LangChain, any function can be bound as a tool.
+
+```python
+def multiply(a: int, b: int) -> int:
```

Comment:
I believe we usually need a `@tool` decorator (since some code probably assumes that the tool is a runnable) at invocation time

```suggestion
def multiply(a: int, b: int) -> int:
```